memory-viz/src/memory_model.ts

```diff
@@ -54,7 +54,7 @@ export class MemoryModel {
     list_index_sep: number; // Vertical offset for list index labels
     font_size: number; // Font size, in px
     browser: boolean; // Whether this library is being used in a browser context
-    seed: number; // Seed for RoughJS generated shape(s). 0 if wanting randomness, between 1 and 2^31 otherwise.
+    roughjs_config: object; // Configuration object that sets default properties of all shapes
```

Comment:
The main purpose of this configuration object is to pass options to rough.js, it's not just about shape properties. Update this documented (and in all other places where you updated the documentation).